src/store/modules/activity.ts

```diff
@@ -256,11 +290,13 @@ const actions = {
       state.buckets.browser_buckets.length > 0;
     const active_available = state.buckets.afk_buckets.length > 0;
     const editor_available = state.buckets.editor_buckets.length > 0;
+    const android_available = state.buckets.window_buckets[0].includes('android');
```

Comment:
Oh, the android bucket just have a normal bucket type?
That certainly makes things confusing.
Let's just hope that no one has 'android' in their hostname lol.
Can't you have something like ".startsWith('aw-watcher-android')" or something instead?

Reply:
Also, I think you need to check that "window_buckets" has a length of 1 or more before doing that check to avoid errors on desktop.

Reply:
Wouldn't it be easier if you in the "buckets" vuex instead had a "state.buckets.android_buckets" instead of here?
You could then also exclude the android buckets from the usual window buckets.

Reply:
>Oh, the android bucket just have a normal bucket type?

Yeah, bad initial choice on my part. I expect us to change this eventually in some Great Migration (as with currentwindow, afkstatus).

>Let's just hope that no one has 'android' in their hostname lol.
>Can't you have something like ".startsWith('aw-watcher-android')" or something instead?

Fixed.

>Also, I think you need to check that "window_buckets" has a length of 1 or more before doing that check to avoid errors on desktop.

Fixed.

>Wouldn't it be easier if you in the "buckets" vuex instead had a "state.buckets.android_buckets" instead of here?
>You could then also exclude the android buckets from the usual window buckets.

Fixed.